### ISSUE TITLE
Invalid longitude error if bounding box contains a zero

### ISSUE THREAD

**Issue by dbaston** [icesat2py/icepyx#325]
Trying to query with the bounding box `(0, 0, 1, 1)` throws the following error:

```
AssertionError: Invalid bounding box longitudes
```

This is apparently because `np.sign(0) != np.sign(1)`. If I try to nudge the bounding box to avoid the error, with `(1e-6, 1e-6, 1, 1)` I get:

```
icepyx.core.exceptions.NsidcQueryError: An error was returned from NSIDC in regards to your query: ['[1e-08,5.00000001,0.99999999,5.99999999] is not a valid URL encoded bounding_box']
```

**Comment by JessicaS11** [comment]
Hello @dbaston! Thanks for reporting this. We're actually in the process of improving our spatial input handling (#322) to address #321 and several other issues (e.g. for inputs that cross the dateline). @sjh1024, perhaps we could add this as a test case?

I'd have to dig in a bit more to try and figure out why the NSIDC API doesn't like the submitted query - it may be that their API cannot handle the scientific notation for coordinate input. I will investigate this more next week after I return from a conference this week.

**Comment by sjh1024** [comment]
Thank you for reporting this! I will add this test case + discuss a potential fix with Jessica this week.

**Comment by JessicaS11** [comment]
Hello again @dbaston! This issue should be resolved with #322 and #331 (which will be included in the next release of icepyx, likely sometime in early October). Please let us know if you run into this or another issue in the future!
